components/lvgl_esp32_drivers/lvgl_tft/disp_spi.c

```diff
@@ -178,9 +166,22 @@ bool disp_spi_is_busy(void)
 
 static void IRAM_ATTR spi_ready (spi_transaction_t *trans)
 {
-    spi_trans_in_progress = false;
+    disp_spi_send_flag_t flags = (disp_spi_send_flag_t) trans->user;
+
+    if (flags & DISP_SPI_SIGNAL_FLUSH) {
+        lv_disp_t * disp = NULL;
+
+#if LV_VERSION_CHECK(7,0,0)
```

Comment:
```suggestion
#if LVGL_VERSION_MAJOR >= 7
```

There aren't any plans to drop the `_` naming convention and the current condition will break in 8.0.

Reply:
Thanks, I will fix that.